include/api.php

```diff
@@ -337,74 +344,88 @@ function pll_save_post_translations( $arr ) {
  * @api
  * @since 1.5
  *
- * @param array $arr an associative array of translations with language code as key and term id as value
+ * @param int[] $arr An associative array of translations with language code as key and term id as value.
+ * @return void
  */
 function pll_save_term_translations( $arr ) {
 	PLL()->model->term->save_translations( reset( $arr ), $arr );
 }
 
 /**
- * Returns the post language
+ * Returns the post language.
  *
  * @api
  * @since 1.5.4
  *
- * @param int    $post_id
- * @param string $field   Optional, the language field to return ( see PLL_Language ), defaults to 'slug'
- * @return bool|string The requested field for the post language, false if no language is associated to that post
+ * @param int    $post_id Post id.
+ * @param string $field   Optional, the language field to return ( @see PLL_Language ), defaults to 'slug'.
+ * @return string|false The requested field for the post language, false if no language is associated to that post.
  */
 function pll_get_post_language( $post_id, $field = 'slug' ) {
 	return ( $lang = PLL()->model->post->get_language( $post_id ) ) ? $lang->$field : false;
 }
 
 /**
- * Returns the term language
+ * Returns the term language.
  *
  * @api
  * @since 1.5.4
  *
- * @param int    $term_id
- * @param string $field   Optional, the language field to return ( see PLL_Language ), defaults to 'slug'
- * @return bool|string The requested field for the term language, false if no language is associated to that term
+ * @param int    $term_id Term id.
+ * @param string $field   Optional, the language field to return ( @see PLL_Language ), defaults to 'slug'.
+ * @return string|false The requested field for the term language, false if no language is associated to that term.
  */
 function pll_get_term_language( $term_id, $field = 'slug' ) {
 	return ( $lang = PLL()->model->term->get_language( $term_id ) ) ? $lang->$field : false;
 }
 
 /**
- * Returns an array of translations of a post
+ * Returns an array of translations of a post.
  *
  * @api
  * @since 1.8
  *
- * @param int $post_id
- * @return array an associative array of translations with language code as key and translation post_id as value
+ * @param int $post_id Post id.
+ * @return int[] An associative array of translations with language code as key and translation post id as value.
```

Comment:
So it is not really an `int[]`, is it?

Reply:
Values are only integers. This is waht the static analysis checks wehne accessing a value.